sitestats/sitestats-impl/src/java/org/sakaiproject/sitestats/impl/chart/ChartServiceImpl.java

```diff
@@ -1292,6 +1293,26 @@ private Comparable getStatValue(Stat s, String fieldCode, Class periodGrouping)
 					}
 				}
 				return action;
+			}else if(fieldCode.equals(StatsManager.T_PAGE)) {
+				if(s instanceof LessonBuilderStat) {
+					String resName = ((LessonBuilderStat)s).getPageTitle();
+					return resName != null ? resName : msgs.getString("page_unknown");
+				}else{
+					return "";
+				}
+            }else if(fieldCode.equals(StatsManager.T_PAGE_ACTION)) {
+				String action = "";
+				if(s instanceof LessonBuilderStat) {
+					String refAction = ((LessonBuilderStat) s).getPageAction();
```

Comment:
I agree `StringUtils.trimToEmpty(refAction)` is nicer, but this copies the existing pattern of the surrounding code.